libs/input-mapping/tests/Functional/DownloadFilesRedshiftTest.php

```diff
@@ -1,64 +1,112 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Keboola\InputMapping\Tests\Functional;
 
 use Keboola\Csv\CsvFile;
 use Keboola\InputMapping\Configuration\File\Manifest\Adapter;
 use Keboola\InputMapping\Reader;
+use Keboola\InputMapping\Staging\AbstractStrategyFactory;
+use Keboola\InputMapping\Staging\NullProvider;
+use Keboola\InputMapping\Staging\ProviderInterface;
+use Keboola\InputMapping\Staging\Scope;
 use Keboola\InputMapping\Staging\StrategyFactory;
 use Keboola\InputMapping\State\InputFileStateList;
 use Keboola\StorageApi\Client;
+use Keboola\StorageApiBranch\ClientWrapper;
+use Keboola\StorageApiBranch\Factory\ClientOptions;
+use Keboola\Temp\Temp;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\NullLogger;
+use SplFileInfo;
 use Symfony\Component\Finder\Finder;
 
-class DownloadFilesRedshiftTest extends DownloadFilesTestAbstract
+class DownloadFilesRedshiftTest extends TestCase
```

Comment:
Tohle je proc? Vadi necemu to clearovani souboru v `DownloadFilesTestAbstract` ?

Reply:
přesně tak, byla to přičina nestabilních testů - DownloadFilesAdaptiveTest a DownloadFiles test běží paralelně s DownloadFilesRedshiftTest a DownloadFilesAbsWorkspaceTest. tím, že byly derived z DownloadFilesTestAbstract  tak to zbytečně clearovalo soubory a pak failovaly testy DownloadFilesAdaptiveTest a DownloadFiles .